js/dataframe/Index.svelte

```diff
@@ -99,6 +99,7 @@
 		{col_count}
 		{value}
 		{headers}
+		on:change={(e) => (value = e.detail)}
```

Comment:
Ah!!!

Reply:
yeee

Reply:
very upsetting that it took me so long to find it tbh. i guess you often miss the obvious stuff.

Reply:
I spent a couple hours trying different things so I feel like you got it pretty fast 😄